README.md

```diff
@@ -151,6 +153,10 @@ experiment = Experiment.new(name="SDK EXP", description="new experiment test for
 client.create(experiment)
 ```
 
+Note: All timestamps on the new experiment must be in Unix Timestamp (Represents time as the number of seconds since the Unix epoch). 
+Additionally, when adding data to a new experiment, all timestamps must be between `start_time` and `end_time` of the experiment to be valid.
```

Comment:
Should we move this section to the `Experiment` section in the validation document?